services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: str
+) -> QuerySet[MovieSession]:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie=movie
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(
+        movie_session_id: str = None
+) -> QuerySet[MovieSession]:
+    if movie_session_id:
+        return MovieSession.objects.get(id=movie_session_id)

```

Comment:
The `get_movie_session_by_id` function uses `get()` to retrieve a `MovieSession` by ID. If the ID does not exist, this will raise a `DoesNotExist` exception. Consider handling this exception to provide a more user-friendly error message.